src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -0,0 +1,253 @@
+        </v-col>
+        <v-col>
+          <p v-if="isSame(person.address.mailingAddress, person.address.deliveryAddress)"
+            class="peoples-roles-delivery-address">Same as Mailing Address
+          </p>
+          <base-address v-else class="peoples-roles-delivery-address" :address="person.address.deliveryAddress"/>
+        </v-col>
+        <v-col>
+          <div v-if="person.roles.length>0">
+            <v-col v-for="(role, index) in person.roles" :key="index" class="col-roles">
+              <span>{{role}}</span>
+            </v-col>
+          </div>
+          <div v-else>
+             <v-icon color="rgb(232, 156, 48)" small>mdi-alert</v-icon>
+             <span class="warningText">Add Role</span>
+          </div>
+        </v-col>
+
+        <!-- Actions Column -->
+        <v-col sm="1" v-if="!isSummary">
+          <div class="actions">
+            <span class="edit-action">
+              <v-btn small text color="primary"
+                :id="'person-' + person.id + '-change-btn'"
+                @click="emitPersonInfo(index)"
+              >
+                <v-icon small>mdi-pencil</v-icon>
+                <span>Edit</span>
+              </v-btn>
+            </span>
+
+            <!-- more actions menu -->
+            <span>
+              <v-menu offset-y>
+                <template v-slot:activator="{ on }">
+                  <v-btn text small color="primary" class="actions__more-actions__btn" v-on="on">
+                    <v-icon>mdi-menu-down</v-icon>
+                  </v-btn>
+                </template>
+                <v-list class="actions__more-actions">
+                  <v-list-item @click="confirmRemove(index)">
+                    <v-list-item-title><v-icon>mdi-delete</v-icon>Remove</v-list-item-title>
+                  </v-list-item>
+                </v-list>
+              </v-menu>
+            </span>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Mixins, Emit } from 'vue-property-decorator'
+
+// Components
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+// Dialogs
+import { ConfirmRemoveDialog } from '@/components/dialogs'
+
+// Mixins
+import CommonMixin from '@/mixins/common-mixin'
+
+// Interfaces
+import { OrgPersonIF } from '@/interfaces'
+
+@Component({
+  components: {
+    BaseAddress,
+    ConfirmRemoveDialog
+  }
+})
+export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
+  // Store Properties
+  @Prop({ default: '' })
+  private personList: Array<OrgPersonIF>
+
+  @Prop({ default: false })
+  private showErrorSummary: boolean
+
+  @Prop({ default: false })
+  private isSummary: boolean
+
+  // Local Properties
+  readonly tableHeaders: Array<string> = ['Name', 'Mailing Address', 'Delivery Address', 'Roles']
```

Comment:
Can i do this a better way? I wanted to make the table somewhat dynamic for future expansion/changes, so took this approach to hardcoding all them into the template. Any other ideas here that are simple?

Reply:
This is pretty nice I think.

You could specify the array right in the template but I'm not sure that's cleaner.

We might decide that all constants (incl. readonly) should be all caps. (That was a C convention.) Thoughts?

Reply:
I'm definitely ok with it as is, i will add this to the list of future code improvements tho!